roc/log_test.go

```diff
@@ -0,0 +1,100 @@
+package roc
+
+import (
+	"io/ioutil"
+	"log"
+	"testing"
+	"time"
+)
+
+const defaultLogLevel LogLevel = LogError
+
+type testWriter struct {
+	ch chan string
+}
+
+func makeTestWriter() testWriter {
+	return testWriter{
+		ch: make(chan string, 1),
+	}
+}
+
+func (tw testWriter) Write(buf []byte) (int, error) {
+	select {
+	case tw.ch <- string(buf):
+	default:
+	}
+	return len(buf), nil
+}
+
+func (tw testWriter) wait() string {
+	select {
+	case s := <-tw.ch:
+		return s
+	case <-time.After(time.Minute):
```

Comment:
maybe it's nice to have `time.Minute` as a named constant? makes things even clearer

Reply:
Done.